papers/DX_zstring_view.fiets

```diff
@@ -81,6 +142,211 @@ The `substr` function is changed from a single function with a default argument,
 
 `remove_suffix` is implementable with `substr` with two parameters, but exposes a more limited interface, as it always retains the start. It always removes the null termination guarantee and should return a `string_view`.
 
+# Bikeshedding
+
+Null-terminated string view types are typically named `zstring_view` (N3921, P3081, GSL) or `cstring_view` (P1402). `cstring_view` follows the `std::string::c_str()` nomenclature while `zstring_view` has some establishment and recognizability.
```

Comment:
I lean slightly to the `std::cstring_view` spelling, however, I am impartial. I tried to do the `std::zstring_view` side justice in this section.